simulator/mips/t/mips32_branch_test.cpp

```diff
@@ -597,6 +682,11 @@ TEST_CASE( "MIPS32_instr: jalr to 2nd byte (round up to 4th)")
     instr.execute();
     CHECK( instr.get_new_PC() == 4);
     CHECK( instr.get_v_dst() == instr.get_PC() + 8);
+
+    CHECK( !instr.is_branch() );
+    CHECK( !instr.is_direct_jump() );
+    CHECK( instr.is_indirect_jump() );
+    CHECK( !instr.is_likely_branch() );
```

Comment:
I think we may leave these only in "leading" tests, i.e. where disassembly is tested.